starwars/src/App.js

```diff
@@ -29,11 +31,22 @@ class App extends Component {
       });
   };
 
+  displayCharacter = (event) => {
+    event.preventDefault();
+    console.log(event.target.className)
+  }
+
   render() {
     return (
-      <div className="App">
-        <h1 className="Header">React Wars</h1>
-      </div>
+      <div className="container">  
+        <h1>Choose Your Destiny</h1>    
+        <div className="buttons">
+          {this.state.starwarsChars.map((char,i) => <Buttons displayCharacter={this.displayCharacter.bind(this)} char={char} key={i} />)}  
```

Comment:
If you use an arrow function, it automatically binds to the class so you don't need to use .bind here 😊